index-new.d.ts

```diff
@@ -0,0 +1,9 @@
+declare module "eris" {
```

Comment:
Make it callable, and maybe(?) use `namespace` instead of `module` since that's the preferred terminology [according to TypeScript](https://www.typescriptlang.org/docs/handbook/modules.html).